test/GenerateProgrammaticPipelineFromConfig/TestAsset/expected/config/autoload/programmatic-pipeline.global.php

```diff
@@ -35,6 +36,9 @@
             // Override the following to use an alternate "not found" delegate.
             NotFoundDelegate::class => NotFoundDelegateFactory::class,
```

Comment:
this still causes `An alias by the name "zendexpressivedelegatenotfounddelegate" or "Zend\Expressive\Delegate\NotFoundDelegate" already exists` but don't know if I can remove it or have to change to something else @weierophinney can you help?